core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -124,7 +132,37 @@ public synchronized List<Result> scan(Scan scan) throws CrudException {
 
     @Override
     public synchronized Scanner getScanner(Scan scan) throws CrudException {
-      return super.getScanner(scan);
+      Scanner scanner = super.getScanner(scan);
```

Comment:
Are these changes for fixing bugs? Or, are they related to the batch thing?

Reply:
As discussed, I'll backport the change to the release branches. Thanks!